aws-api-appsync/src/main/java/com/amplifyframework/api/aws/TemporalDeserializers.java

```diff
@@ -30,53 +34,73 @@
  * AppSync temporal scalars</a>.
  */
 public final class TemporalDeserializers {
+    private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-datastore");
```

Comment:
```suggestion
    private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-api");
```

I think I gave you the wrong namespace for Logging when chatting on Discord, this should be `aws-api`.